lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -581,6 +581,60 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		}
 		return true;
 	});
+	/**
+	* Sorts an array in-place.
+	*
+	* @private
+	* @name sort
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} compareFcn - comparison function
+	* @throws {TypeError} `this` must be a typed array
+	* @throws {TypeError} first argument must be a function
+	* @returns {TypedArray} sorted array
+	*/
+	setReadOnly( TypedArray.prototype, 'sort', function sort( compareFcn ) {
+		var minIdx;
+		var minVal;
+		var val;
+		var buf;
+		var i;
+		var j;
+
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError( format( 'invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0] ], CTOR_NAME ) );
+		}
+		if ( arguments.length === 0 ) {
+			compareFcn = function defaultCompareFcn( a, b ) {
+				if ( a > b ) {
+					return 1;
+				}
+				if ( a < b ) {
+					return -1;
+				}
+				return 0;
+			};
+		} else if ( !isFunction( compareFcn ) ) {
+			throw new TypeError( format( 'invalid argument. First argument must be a function. Value: `%s`.', compareFcn ) );
+		}
+		buf = this._buffer;
+		for ( i = 0; i < this._length-1; i++ ) {
```

Comment:
Rather than a hand-rolled sort implementation, just copy to a temporary buffer (e.g., a plain array), use the built-in sort, and then copy back to the original buffer. That will be `2N + N log N`, which is still better than `N^2`.

Reply:
Ok, I'll update it